torch/_dynamo/variables/nn_module.py

```diff
@@ -1055,24 +1081,36 @@ def call_function(
         # Check if we can short circuit nn.Module._call_impl to the forward
         # method.  NB - This is done to reduce the compile time of Dynamo.
         if (
-            istype(mod.__call__, types.MethodType)
-            and istype(mod._call_impl, types.MethodType)
-            and mod.__call__.__func__ is unpatched_nn_module_call
-            and mod._call_impl.__func__ is unpatched_nn_module_call_impl
+            istype(mod.__call__, types.MethodType)  # type: ignore[operator]
+            and istype(mod._call_impl, types.MethodType)  # type: ignore[attr-defined]
```

Comment:
Would be the case if _call_impl was undefined. Would you want to raise an error or just skip the if case, type ignore could be removed with hasattr if later